tools/merge_config_paramsters/README.md

```diff
@@ -0,0 +1,32 @@
+
+# Merge model config and parameters
+
+Integrate the model configuration and model parameters into one file. 
+The merged file is used in our capi forward predict program.
+
+## Demo
+
+This applies to all PaddlePaddle v2 models, we show a demo of mobilenet.
+
+### Step 1: Prepartions
+
+**Model Config :** [Mobilenet model config](../../models/mobilenet.py).    
+**Model Parameters:** [Mobilenet model param pretrained on flower102 download](https://pan.baidu.com/s/1geHkrw3) 
+
+### Step2: Merge 
+
+Run the following code
+
+```
+from paddle.util.merge_model import merge_v2_model
```

Comment:
A typo: from paddle.**utils**.merge_model import merge_v2_model

Reply:
ok, i will fix it ASAP